src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -165,20 +169,28 @@ private static String getApiToken() {
             urlParameters.add(new BasicNameValuePair("audience", audienceUrl));
             post.setEntity(new UrlEncodedFormEntity(urlParameters));
             HttpResponse response = client.execute(post);
+            // Read response code/entity.
             int code = response.getStatusLine().getStatusCode();
-            if (code >= 300) LOG.error("Could not get Auth0 API token");
-            String token = EntityUtils.toString(response.getEntity());
-            LOG.info(token);
-            Auth0AccessToken auth0AccessToken = mapper.readValue(token, Auth0AccessToken.class);
-            LOG.info("Scope: {}", auth0AccessToken.scope);
-            LOG.info("Token type: {}", auth0AccessToken.token_type);
-            // Cache token for later use.
-            cachedToken = auth0AccessToken;
-            return cachedToken.access_token;
+            responseString = EntityUtils.toString(response.getEntity());
+            if (code >= 300) {
+                LOG.error("Could not get Auth0 API token {}", responseString);
+                throw new IllegalStateException("Bad response for Auth0 token");
+            }
         } catch (IllegalStateException | URISyntaxException | IOException e) {
```

Comment:
Is IOException still needed here?

Reply:
Yea, it's thrown by the few lines pertaining to the request/response entity in this block.